src/main/scala/se/nimsa/sbx/util/ReverseAnonymizationFlow.scala

```diff
@@ -67,7 +67,7 @@ class ReverseAnonymizationFlow() extends GraphStage[FlowShape[DicomPart, DicomPa
             // FIXME: remove println
             println(">>>> grabbed meta, isAnon: " + isAnonymized)
             println(">>>> grabbed meta, canDoReverse: " + canDoReverseAnon)
-            pull(in)
+            push(out, metaPart)
```

Comment:
True. But maybe it is reasonable that flows that handle `DicomPart`s do not make assumptions on what implementations of this interface there may be? I´m thinking that unless such stages perform some kind of filtering a reasonable standard should be to pass on unknown parts.